src/q7/crypto-MAT2450/eplcrypto.sty

```diff
@@ -0,0 +1,52 @@
+\NeedsTeXFormat{LaTeX2e}[1994/06/01]
+\ProvidesPackage{eplcrypto}[2020/01/16 EPL crypto package for cryptography-related courses]
+
+% Encryption
+\newcommand{\A}{\mathcal{A}}
+\newcommand{\Adv}{\ensuremath{\mathcal{A}}}
+\renewcommand{\K}{\mathcal{K}}
+\renewcommand{\C}{\mathcal{C}}
+\newcommand{\M}{\mathcal{M}}
+\newcommand{\D}{\mathcal{D}}
+\renewcommand{\O}{\mathcal{O}}
+\newcommand{\G}{\mathcal{G}}
+\DeclareMathOperator{\Gen}{\mathsf{Gen}}
+\DeclareMathOperator{\Enc}{\mathsf{Enc}}
+\DeclareMathOperator{\Dec}{\mathsf{Dec}}
+\DeclareMathOperator{\Mac}{\mathsf{Mac}}
+\DeclareMathOperator{\Vrfy}{\mathsf{Vrfy}}
+\DeclareMathOperator{\Com}{\mathsf{Com}}
+\DeclareMathOperator{\Open}{\mathsf{Open}}
+\DeclareMathOperator{\Sign}{\mathsf{Sign}}
+\DeclareMathOperator{\Sig}{\mathsf{Sig}}
+\DeclareMathOperator{\PRG}{\mathsf{PRG}}
+\DeclareMathOperator{\F}{\mathsf{F}}
+\DeclareMathOperator{\f}{\mathsf{f}}
+\DeclareMathOperator{\sprp}{\mathsf{sprp}}
+\DeclareMathOperator{\adv}{\mathsf{adv}}
+\DeclareMathOperator{\Gr}{\mathsf{Gr}}
+\DeclareMathOperator{\Ima}{\mathsf{Im}}
+
+
+% Experiments
+\newcommand{\PrivK}{\mathsf{PrivK}}
+\newcommand{\PrivKeav}[1][\A,\Pi]{\PrivK_{#1}^\mathsf{eav}}
+\newcommand{\PrivKmult}[1][\A,\Pi]{\PrivK_{#1}^\mathsf{mult}}
+\newcommand{\PrivKcpa}[1][\A,\Pi]{\PrivK_{#1}^\mathsf{cpa}}
+\newcommand{\PrivKcca}[1][\A,\Pi]{\PrivK_{#1}^\mathsf{cca}}
+\newcommand{\MacForge}{\mathsf{MacForge}}
+\newcommand{\EncForge}{\mathsf{EncForge}}
+\newcommand{\Sigforge}{\mathsf{Sig-forge}}
+
+% Functions
+\newcommand{\negl}{\ensuremath{\epsilon}}
+\newcommand{\define}{\ensuremath{:=}}
```

Comment:
```suggestion
\newcommand{\define}{\ensuremath{\coloneqq}}
```

You will need `mathtools` to be able to use this, I'm not sure it is automatically loaded (it is part of `src/eplcommon.sty`, though I'm not convinced it is loaded here), but that way you get a horizontally symmetrical symbol which looks neater.

Reply:
Actually, looking further ahead, it seems you'd need `\mathtoolsset{centercolon}`, which automatically makes the definition symbol centered (or at least I think, I haven't checked) (since the command is not used consistently throughout the document).

Reply:
Intéressant, en mettant `\mathtoolsset{centercolon}` dans eplcrypto ou dans le fichier principal, la compilation part en quasi boucle infinie (même après une minute, la première compilation n'est pas finie, alors que sans, ça prend juste 5-10 secondes). Je ne sais pas à quoi c'est dû.
J'applique la solution la plus simple du coup : éliminer les `:=`.